editor/editor_node.h

```diff
@@ -169,7 +170,7 @@ class EditorNode : public Node {
 		RUN_PLAY_CUSTOM_SCENE,
 		RUN_SETTINGS,
 		RUN_USER_DATA_FOLDER,
-		RUN_RELOAD_CURRENT_PROJECT,
+		RELOAD_CURRENT_PROJECT,
```

Comment:
Ah I had missed that `FILE_CLOSE_ALL_AND_RUN_RELOAD_CURRENT_PROJECT` was matching a pre-existing `RUN_RELOAD_CURRENT_PROJECT`. It seems like contributors over time have all prefixed their new actions with `RUN_` without understanding that the original `RUN_` actions are the actions to actually run the project/a scene.

So `RUN_SETTINGS`, `RUN_USER_DATA_FOLDER`, `RUN_PROJECT_MANAGER` are all misnomers. But this is material for a different PR refactoring that code.